README.md

```diff
@@ -1,4 +1,4 @@
-# Dapr - Any language, any framework, anywhere
+# Dapr - any language, any framework, anywhere
```

Comment:
This was intentional, this should not be changed.

Reply:
thanks! revert.